src/Card/CardImage.component.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react"
+import styled from "styled-components"
+
+type ImageSize =
+  | "mini"
+  | "small"
+  | "medium"
+  | "large"
+  | "big"
+  | "huge"
+  | undefined
+
+type ImageSpaceDirection = "left" | "right" | "top" | "bottom" | undefined
+
+export interface ImageProps extends React.HTMLAttributes<HTMLImageElement> {
+  /**
+   * Float card left or right
+   */
+  floated?: "left" | "right"
+  /**
+   * Rounded corners
+   */
+  rounded?: boolean
+  /**
+   * Circular image
+   */
+  circular?: boolean
+  /**
+   * Size of the image
+   */
+  size?: ImageSize
+  src: string
+  /**
+   * Add spacing to left | right | top | bottom
+   */
+  space?: ImageSpaceDirection
+}
+
+const getSize = (size: ImageSize): string => {
```

Comment:
Would highly encourage looking into the object look up method. I prefer it over switch statements. They have less computational overhead as well. 

```jsx
const retrunSomething =(arg) => {
  const sizes = {
    small: '150px'
  }
  return sizes[arg]
}
```